CMakeLists.txt

```diff
@@ -248,6 +249,7 @@ if(BUILD_TESTING)
     ENV ${memory_tools_test_env_vars}
   )
   if(TARGET test_error_handling_helpers)
+    target_include_directories(test_error_handling_helpers PUBLIC include)
```

Comment:
For consistency, is it possible to use `$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"` like you did earlier in the file?

Reply:
The generator expression is only needed for the interface target. In this case just using `include` is simpler and sufficient and replacement of `include_directories(include)`.